examples/blob-maker/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "blob-maker",
+  "version": "0.1.0",
+  "licence": "MIT",
+  "scripts": {
+    "dev": "npx supervisor server.js",
```

Comment:
You can just call `supervisor` here directly since it's installed with the application.